Content.Server/_NF/Storage/AnchorableStorageSystem.cs

```diff
@@ -48,31 +43,29 @@ private void OnAnchorStateChanged(Entity<AnchorableStorageComponent> ent, ref An
         }
 
         // Eject any sapient creatures inside the storage.
-        // Hack: does not recurse down into bags in bags.
+        // Does not recurse down into bags in bags - player characters are the largest concern, and they'll only fit in duffelbags.
         if (!TryComp(ent.Owner, out StorageComponent? storage))
             return;
-        List<EntityUid> entsToRemove = new();
-        foreach (var storedItem in storage.StoredItems.Keys)
-        {
-            if (HasComp<MindContainerComponent>(storedItem) || HasComp<PseudoItemComponent>(storedItem))
-                entsToRemove.Add(storedItem);
-        }
+
+        var entsToRemove = storage.StoredItems.Keys.Where(storedItem =>
+            HasComp<MindContainerComponent>(storedItem)
+            || HasComp<PseudoItemComponent>(storedItem)).ToList();
```

Comment:
very minor nitpick but if you're splitting it up, .ToList() should be in a new line
everything else looks good

Reply:
Correct me if I'm wrong but you submitted this as a one-liner.

Reply:
> Correct me if I'm wrong but you submitted this as a one-liner.

yes but if it's not a one liner here then it should be a four liner